etcd/map.jinja

```diff
@@ -1,6 +1,6 @@
 {% set server = salt['grains.filter_by']({
     'Debian': {
-        'pkgs': ['etcd', 'python-etcd'],
```

Comment:
We are trying to have cases that are production usable in formulas and pkg is no longer that option, so there is no need to have it there.